### ISSUE TITLE
Empty password not allowed during setting up locally

### ISSUE THREAD

**Issue by arunikayadav42** [anitab-org/portal#485]
## Description
As a contributor,
I should be able to setup the local database with an empty password too,
so that I can finish up setting up the project locally without any errors.

### Update [Required]
- [ ] Add support for such functionality

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
[1] hours

**Comment by sammy1997** [comment]
Hey @arunikayadav42 ! Why do you want to add this feature? Can you explain it a bit?

**Comment by arunikayadav42** [comment]
Just because I was trying to set it up locally and I did not initially enter any passwords for the user. I got an error. Tried to remove the Password field from the database settings but still it did not work, so I had to move on to make another user without an empty password. So, just thought it could be good to allow empty passwords. or if we are not allowing for empty passwords, maybe it can be good to make an addition in the readme.
